### DISCUSSION TITLE
#1387 Issue - How to specify different directory - Save item command

### DISCUSSION THREAD

**Discussion by mbakstars** [hluk/CopyQ#2143] [Q&A]
Please forgive me for bringing up an older ticket. 

But for issue (Is there a way to save/export images #1387) and [application command](https://github.com/hluk/copyq-commands/blob/master/Application/save-item-clipboard-to-file.ini)

Is there way to specify a different directory. I am not overly familiar with the code language. I know the main command would not be modified to allow the best cross platform compatibility. But asking how I might be able modify it for my particular systems; mostly windows 10 and 11.

**Comment by hluk** [comment]
I think the default directory can be changed by setting `var defaultFileName = 'untitled'` to a full path - but use `/` as path/directory separators instead of `\`. Example:

```js
var defaultFileName = 'C:/Documents/untitled'
```
